### PR TITLE
Fixed the paramedic not having access on NSS Kerberos Departures cycled airlock

### LINKED ISSUE #22861: Paramedic doesn't have access to Kerberos Departures cycled airlock
**Issue Description**:
Title

**What did you expect to happen**: 
Be able to take a shorter path

**What happened instead**:
_`Access denied.`_

**Why is this bad/What are the consequences**:
Paramedic is one of the station roles that really benefits from fast response times, hence they have access to most other maintenance-space cycled airlocks

**Steps to reproduce the problem**:
Beep the button on the cycled airlock in bottom-left of Kerberos's Departures

**When did the problem start happening**:
`*shrug`

**Extra information**:
Thanks, have a Paradise day!

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Fixed the departures airlock access and moved a few things around

 Fixes #22861 

## Why It's Good For The Game
Makes the paramedic have faster response in Departures In case someone dies in spess

## Images of changes
 Before

 After
Screenshot 2023-12-21 212804

## Testing
 Spawned as a paramedic and tested that I got access and I did

## Changelog
:cl:
Add: added a airlock helper for the airlock
Tweak: made the room a bit bigger
/:cl:

### CODE REVIEW

**Review by Sirryan2002 — APPROVED** [review]

**Review by Henri215 — COMMENTED** [review]
image

Remove those two dark tiles since they are not close to a wall anymore.

File: _maps/map_files/Delta/delta.dmm [hunk 1]
```diff
@@ -71997,8 +71991,12 @@
 /turf/simulated/floor/plasteel,
 /area/station/maintenance/apmaint)
 "dTh" = (
-/obj/machinery/door/airlock/maintenance,
-/obj/effect/turf_decal/stripes/line,
+/obj/effect/turf_decal/delivery,
+/obj/machinery/camera{
+	c_tag = "Virology south";
```
Comment (lewcc) [inline _maps/map_files/Delta/delta.dmm:71996]: Is this that same departures camera? Looks like it might need to be re-labeled (and re-networked).

Comment (Fluffyzack21) [reply _maps/map_files/Delta/delta.dmm:71996]: fixed

**Review by lewcc — APPROVED** [review]
Changes look good

**Pull request merged**